bootstrap/src/sun/nio/ch/lincheck/EventTracker.java

```diff
@@ -32,6 +32,7 @@ public interface EventTracker {
     void beforeNewObjectCreation(String className);
     void afterNewObjectCreation(Object obj);
 
+    void updateSnapshotWithEagerTracking(Object[] objs);
```

Comment:
Please add a newline to separate `updateSnapshotWithEagerTracking` from `beforeReadXXX` methods.

Reply:
done